lib/model/migrations/20240806-01-add-userpreferences.js

```diff
@@ -0,0 +1,21 @@
+// Copyright 2024 ODK Central Developers
+// See the NOTICE file at the top-level directory of this distribution and at
+// https://github.com/getodk/central-backend/blob/master/NOTICE.
+// This file is part of ODK Central. It is subject to the license terms in
+// the LICENSE file found in the top-level directory of this distribution and at
+// https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+// including this file, may be copied, modified, propagated, or distributed
+// except according to the terms contained in the LICENSE file.
+
+const up = (db) => db.schema.createTable('userpreferences', (prefs) => {
+  prefs.integer('userId').notNull();
```

Comment:
I can't explain it but I feel uncomfortable with this being called `userId` and then it being a FK to an `actorId`, except I guess it's the only ID on the users table, and these user preferences are only going to apply to the subset of Actors that are Users. Does it make me feel better to call it `actorId` and have it FK directly to the `actors` table? Not really. Would any other actors have user preferences? Uhh, I currently can't think far enough ahead to know what that means. 

Maybe the way it is is as `userId` is really the best way.